deployments/charts/kuma/README.md

```diff
@@ -88,9 +88,10 @@ A Helm chart for the Kuma Control Plane
 | cni.image.registry | string | `"docker.io/kumahq"` | CNI image registry |
 | cni.image.repository | string | `"install-cni"` | CNI image repository |
 | cni.image.tag | string | `"0.0.10"` | CNI image tag |
-| cni.imageExperimental | object | `{"repository":"kuma-cni","tag":null}` | use new CNI image (experimental) |
-| cni.imageExperimental.repository | string | `"kuma-cni"` | CNI experimental image repository |
-| cni.imageExperimental.tag | string | `nil` | CNI experimental image tag - defaults to .Chart.AppVersion |
+| cni.test | object | `{"sleepBeforeRunSeconds":0}` | variables used in e2e tests |
```

Comment:
```suggestion
| cni.delayStartupSeconds | int | 0 | delay the startup of the cni (only useful in testing) |
```

Reply:
How about not prefixing it with test. It's just a way to delay the start of the cni (you just explain in the doc that it's only useful in tests.